src/contract_wrappers/registries/security_token_registry_wrapper.ts

```diff
@@ -1265,4 +1279,22 @@ export default class SecurityTokenRegistryWrapper extends ContractWrapper {
       'Msg sender must be owner',
     );
   };
+
+  private checkRegisterTickerRequirements = async (ticker: string, owner: string) => {
+    assert.isETHAddressHex('owner', owner);
+    assert.assert(ticker.length > 0 && ticker.length <= 10, 'Bad ticker, must be 1 to 10 characters');
+    assert.assert(
+      await this.isTickerAvailable({
+        ticker,
+      }),
+      'Ticker is not available',
+    );
+
+    // Check poly token allowance
```

Comment:
Here you're checking for balance, not allowance, so the comment and error message should reflect that